docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1798 @@
+{
```

Comment:
<blockquote>Moreover, the response variable can also be a discrete count, and ordered</blockquote><p> </p><p>If a variable is a count then it's in a ratio scale (nominal &lt; ordinal &lt; interval &lt; ratio), which is more powerful than ordered, so it's not needed to mention that.</p><p> </p><blockquote>The distribution of ordinal responses may be nonnormal, particularly if very low or high values are infrequently chosen.</blockquote><p> </p><p>The most important reason why the distribution will be nonnormal is because the response variable is not continuous. Still, there are cases where the variable is not continuous but a normal distribution makes sense (imagine a count with a bell-shaped distribution). But is not usually the case in ordinal variables because the number of response levels is not usually high</p><p> </p><blockquote>(1) cumulative link, and (2) sequential link.</blockquote><p> </p><p>I would call them models or families, but not link, which makes me think of link functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks for the details. I agree with all of this.</p><p> </p>